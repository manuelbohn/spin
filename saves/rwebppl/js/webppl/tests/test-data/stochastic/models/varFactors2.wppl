var model = function() {
  if (flip(.5)) {
    factor(Math.log(.5));
    return 1;
  } else {
    return 2;
  }
};
