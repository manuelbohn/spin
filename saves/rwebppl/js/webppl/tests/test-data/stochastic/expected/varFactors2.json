{
  "hist": {
    "1": 0.33333333,
    "2": 0.66666666
  }
}
